osi_common.proto

```diff
@@ -569,6 +687,19 @@ message BaseMoving
     // The polygon is defined counter-clockwise.
     //
     repeated Vector2d base_polygon = 7;
+
+    // Sub-divisions of the overall bounding box of the \c BaseMoving object.
+    //
+    // The bounding box sections can include side mirrors, cargo, etc. for 
+    // vehicles, as well as body-part sections for pedestrians.
+    //
+    // \note When one or more \c BoundingBox s are associated to a 
+    // \c BaseMoving , the expectation is that all sections are contained
+    // within the bounds of the \c #dimension . Currently there several accepted
+    // exceptions to this rule, specifically the sub-sections for cargo, vehicle
+    // side mirrors, and doors not in the their closed position.
```

Comment:
Proposal: Move the text for the "exceptional cases" to the enum field where they would be used from.